week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,13 @@ let writers = [
   }
 ];
 
+writers.forEach(writer => console.log(
+ `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`));
+
+
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+
+console.log(writers.filter(writer => writer.alive).map(writer => `${writer.firstName} ${writer.lastName}`));

```

Comment:
The `map` not necessarily needed as `console.log(writers.filter(writer => writer.alive))` should be enough.